src/constants.ts

```diff
@@ -193,6 +193,8 @@ export const EMAIL_ALREADY_EXISTS_CODE = "email.alreadyExists";
 export const EMAIL_ALREADY_EXISTS_MESSAGE = "email.alreadyExists";
 export const EMAIL_ALREADY_EXISTS_PARAM = "email";
 
+export const BASE_URL = "http://localhost:4000/";
```

Comment:
You can store the `BASE_URL` like this, but if someone starts the server with a `process.env.PORT` other than `4000`, this won't work.